pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/camel/RuntimeDependenciesToAlignTree.java

```diff
@@ -65,15 +65,21 @@ private String parseDependency(String line) {
 
     @Override
     public void trigger() {
-        String filename = extrasPath + "CamelRuntimeDependenciesToAlignTree.txt";
-        log.info("Running RuntimeDependenciesToAlignTree - report is {}", filename);
-        try (PrintWriter file = new PrintWriter(filename, StandardCharsets.UTF_8.name())) {
+        String filename = extrasPath + "DependenciesToAlignTree.txt";
+        String buildFromSourceStatsFileName = extrasPath + "BuildFromSourceStats.txt";
+        log.info("Running DependenciesToAlignTree - report is {}", filename);
+        PrintWriter file = null;
+        PrintWriter buildFromSourceStatsFile = null;
+        try {
```

Comment:
Hi @cunningt ! Could this code be refactored to use multiple items in the [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) ? Then the cleanup code in the finally clause can be removed.

Otherwise the changes look good to me!

Reply:
Thanks for the feedback - moved the finally code into a try-with-resources

Reply:
LGTM! :)